dask_cudf/tests/test_sort.py

```diff
@@ -22,6 +22,7 @@ def test_sort_values(nelem, nparts, by):
     pd.util.testing.assert_frame_equal(got, expect)
 
 
+@pytest.mark.xfail(reason="")
```

Comment:
@quasiben can we relocate the tests to another PR?

Reply:
Yeah, I can do that